package-lock.json

```diff
@@ -0,0 +1,14187 @@
+{
```

Comment:
Why was this committed?

Reply:
Usually, as a practice, we commit the package.lock.json file for tracking errors. if not required, please let us know we will revert the file.

Reply:
We use `yarn` for this project, so if anything it would be a `yarn.lock` file. But this is not required.

Reply:
Removed the package-lock.json file based on the review comment.